lib/assets/blog-article.mst

```diff
@@ -5,9 +5,9 @@
 </head>
 
 <body>
-<header class="extra-header">
```

Comment:
Changing the name everywhere, in one commit, and the unrelated CSS changes in another commit.